code_style/dot-eslintrc.json

```diff
@@ -24,6 +24,20 @@
   },
   "rules": {
     "semi": 2,
-    "no-console": 1
+    "no-console": 1,
+    "no-undef": 0,
```

Comment:
Can we remove this? I think we should use /* global myGlobalVar */ where needed for old Meteor 1.2 projects.